source/help/settings/account-settings.rst

```diff
@@ -216,6 +216,7 @@ Language
 
 Select what language Mattermost displays in the user interface. Options include:
 
+- български - Bulgarian
```

Comment:
Bulgarian should be listed below Turkish
![image](https://user-images.githubusercontent.com/13119842/106201153-06a63580-6186-11eb-8132-ff3082c9bc2e.png)

Reply:
@jasonblais - Thank you. To clarify, would you like to see Bulgarian listed after Turkish in all cases where we list language support throughout the doc set (minus the Admin Guide config setting cases where we list supported languages alphabetically by short name only)? Or would you like to see this change for this page only?

Reply:
Just this page, which is for account settings specifically - hence why it's good to keep the same order with the Account Settings observed in the product.

In other pages, we resort to mostly alphabetical order.

Reply:
Thanks, @jasonblais. Updated.